client/src/commands.ts

```diff
@@ -78,6 +78,7 @@ export function initializeWorkspace(
     try {
       const settings = await pickInitWorkspace();
       const config = vscode.workspace.getConfiguration(EXTENSION_NS);
+      const languageSpecificConfigs = await Promise.all(["typescript", "typescriptreact"].map(async (languageId) => vscode.workspace.getConfiguration("", { languageId })))
```

Comment:
**Note**: Did this because, since language specifics are not namespaced by the extension settings, we couldn't really rely on the other variable since it would return nothing. So I added it as an array in case we need to add more support (JS, JSX) in the future, so this would be very easy to do.

Also, in case we need to change any other language-specific configs in the future we just need to iterate through it again.